README.md

```diff
@@ -96,6 +96,8 @@ For static routing, please use the following variables:
 |------|-------------|:----:|:-----:|:-----:|
 | route_priority | The priority for static routes being added to VPC | int | 1000 | no |
 | remote_subnet | List of IP ranges that will be accessible over this VPN gateway. A route is created for each range in this list with next hop as VPN tunnels | list | - | yes |
+| local_traffic_selector | Local traffic selector to use when establishing the VPN tunnel with peer VPN gateway. Value should be list of CIDR formatted strings and ranges should be disjoint | list | ["0.0.0.0/0"] | no |
```

Comment:
This list is auto-generated, if you want these descriptions they should go in the variable description: https://github.com/terraform-google-modules/terraform-google-vpn/pull/4/files#diff-c9ac8098c5ea9d3e6a9a596ff0c512a4R45

@andreyk-code

Reply:
Ah didn't realize that. I can update the descriptions there. Looks like the README tables haven't been generated in a while since  many of the descriptions don't match.